starters/nextjs-starter-ts/components/grid.tsx

```diff
@@ -22,7 +22,7 @@ const GridItem = ({ href, imgSrc, altText, tags, title }: Props) => {
         <Link passHref href={href}>
           <div className="relative flex-shrink-0 h-40 cursor-pointer hover:border-indigo-500 border-2s">
             {imgSrc != null ? (
-              <img
+              <Image
```

Comment:
Changing to use the Next Image component means we need to provide the height and width props here as well as configure the next config with all the expected image src hosts. I don't think that's feasible given this comes from a metadata field. 

This currently breaks the app as is when an image thumbnail is provided, should probably leave it as an `img`
![image](https://github.com/pantheon-systems/pantheon-content-cloud-sdk/assets/87580113/e6555180-3269-46c1-acc5-6b836765274d)